sqlalchemy_continuum/transaction.py

```diff
@@ -120,7 +122,8 @@ class Transaction(
             __versioning_manager__ = manager
 
             id = sa.Column(
-                sa.types.BigInteger,
+                sa.types.Integer,
```

Comment:
Is there some reason you changed this to Integer instead of BigInteger? Integer is pretty limited considering this table saves transactions and even in moderately big services there can be more transactions stored than Integer type could handle.